pkg/unpack/oci.go

```diff
@@ -48,34 +48,33 @@ type OCI struct {
 
 type OCIOpt func(*OCI)
```

Comment:
In fact I just saw that this `type` is never used (was before the changes made in this PR). Maybe we could remove it?

Reply:
Nice catch, I will put up a new PR!